oolushola/Banka
# #[165169634] Unit test user signup endpoint

#### What does this PR do?
Have a signup user endpoint in node/express to register a new user for the banka app

#### Description of Task to be completed?
Unit test the following endpoints with the appropriate response status code and validations

- GET: /

- POST:/api/v1/auth/register

#### How should this be manually tested?
After cloning the repo, cd into it and from the terminal RUN 'npm run test' to check for all assertions whether it passes or fails.

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#165169634](https://www.pivotaltracker.com/story/show/165169634)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
